### ISSUE TITLE
Seeing 'AND' instead of 'Y'

### ISSUE THREAD

**Issue by AMN17** [DevangThakkar/wordle_archive#45]
<img width="496" alt="y is showing and" src="https://user-images.githubusercontent.com/98180713/156689499-c3cdd590-3185-442f-9cdf-a9300ad7c740.PNG">
<img width="441" alt="bug in wordle archive" src="https://user-images.githubusercontent.com/98180713/156689514-2d701003-d20b-457d-8df1-e0321fa8d4a7.PNG">

**Comment by mwpatrick** [comment]
Hmm - doing puzzle 147 on IOS 15 on an iPhone 13 Pro
Deliberately guessed a word with a Y as the last letter
Showed up fine - no AND

**Comment by DevangThakkar** [comment]
Yeah, this is very strange. I have no idea why this would happen. Keeping this open in case someone has any idea.
